remediation/fence-agents-remediation/how-it-works.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: How It Works
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 1
+---
+
+## How does Fence Agents Remediation work?
+
+The Fence Agents Remediation (FAR) operator uses a traditional API to remediate unhealthy Kubernetes nodes.
+It is designed to be used with [Node HealthCheck Operator](https://github.com/medik8s/node-healthcheck-operator) (NHC),
+which creates a Custom Resource (CR) for FAR with information about the unhealthy Node.
+The remediation includes rebooting the unhealthy node using a fence agent and evicting workloads from the unhealthy node by resource deletion.
+
+FAR uses a fence agent to fence a Kubernetes node. Generally, fencing is the process of taking unresponsive/unhealthy servers into a safe state and isolating the server.
+Fence agent is a software code that uses an Application Programming Interface (API) call to perform fencing, mostly power-based fencing which enables power-cycling, resetting, or turning off the server.
```

Comment:
Yes, in the end, it is a Python script. But we could mention it differently.
How about just `code` or `script`?
I thought of highlighting that since it  is different than SNR which uses a hardware device (but I might be wrong here because I remember that there is a software reboot command with SNR, right?)
Furthermore, I wanted to be more specific and elaborate on what does fence agent term really means.

Reply:
`application`?

Reply:
SGTM